mantidproject/mantid
# Add ability to ignore 1D arrays for time value loading in LoadNexusLogs.

**Description of work.**
We need to be able to actively load files that contain Time array values but currently we do not know what to do with them so we should just ignore them.

**To test:**
- Run this script
```python
Load(Filename='larmor_array_time_series_mock.nxs',OutputWorkspace='larmor_array_time_series_mock')
```
- If it finishes and you notice two lines of warning in the terminal referencing TimeSeriesProperty then it worked.


Fixes #23911 . 


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
